src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonMetadataServices.Converters.cs

```diff
@@ -110,6 +110,12 @@ public static partial class JsonMetadataServices
         public static JsonConverter<string> StringConverter => s_stringConverter ??= new StringConverter();
         private static JsonConverter<string>? s_stringConverter;
 
+        /// <summary>
+        /// Returns a <see cref="JsonConverter{T}"/> instance that converts <see cref="TimeSpan"/> values.
+        /// </summary>
+        public static JsonConverter<TimeSpan> TimeSpanConverter => s_timeSpanConverter ??= new TimeSpanConverter();
```

Comment:
I was going to add a comment about there needing to be tests exercising the new API, but I couldn't find a test class doing this for the existing `JsonMetadataServices` converters. cc @layomia

Reply:
Since this is being [initialized](https://github.com/dotnet/runtime/blob/0248a1529654fe99bb029b2a77728a0dbf5f0c6a/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.Converters.cs#L75) on start-up, the new tests in this PR exercise this property. When we add corresponding source-gen support for `TimeSpan`, we'll also add tests for that scenario.